pkg/shortnames/shortnames.go

```diff
@@ -149,9 +157,9 @@ const (
 func (r *Resolved) Description() string {
 	switch r.rationale {
 	case rationaleAlias:
-		return fmt.Sprintf("Resolved short name %q to a recorded short-name alias (origin: %s)", r.userInput, r.originDescription)
+		return fmt.Sprintf("Resolve %q->%q (%s)", r.userInput, r.candidates(), r.originDescription)
```

Comment:
It doesn’t make sense to put the whole list of candidates, with possible commas, in quotes.

(Also `Description` is small enough, and the loop building the candidates list seems difficult enough to reuse, that it might be a bit cleaner to inline it.)